### PR TITLE
[PVT#178806766] Non-root Docker images

### PR DESCRIPTION
**Pull request by andyzito**
This is basically just a copy of the work we did in warehouse-land. I am ready to do a test deploy to QA tomorrow.

### CODE REVIEW

File: config/deploy/docker/assets/Dockerfile.open-path-cas.base [hunk 1]
```diff
@@ -68,8 +68,16 @@ RUN mkdir -p app/assets/stylesheets/theme/styles && \
   rm .env && \
   mkdir tmp/pids
 
+RUN chown -R app:app tmp
+RUN touch /etc/timezone
+RUN ln -sf /app/etc-localtime /etc/localtime
+RUN chown app:app /etc/timezone
+
```
Comment (eanders) [inline config/deploy/docker/assets/Dockerfile.open-path-cas.base]: I believe this is missing some items.  In the warehouse this is:
```
RUN touch /etc/timezone
RUN ln -sf /app/etc-localtime /etc/localtime
RUN chown -R app:app tmp
RUN chown -R app:app node_modules
RUN chown -R app:app /usr/local/bundle
RUN chown app:app /etc/timezone
```

**Review by eanders — APPROVED** [review]
@andyzito , I'll defer to @yakloinsteak for a final review, but other than the one note below, (above?) this looks good enough to test a deployment.

**Review by yakloinsteak — APPROVED** [review]
Looks fine enough for a test deployment to me.

File: config/deploy/docker/assets/Dockerfile.open-path-cas.base (hunk 1 above)
Comment (andyzito) [reply config/deploy/docker/assets/Dockerfile.open-path-cas.base]: Good catch, I missed those in the diff cause they came after the PR merge. Thanks!

**Pull request merged**